packages/crayons-core/src/components/timepicker/timepicker.tsx

```diff
@@ -209,6 +206,19 @@ export class Timepicker {
     }
   }
 
+  @Watch('locale')
+  async handleLocaleChange(newLocale) {
+    this.langModule = await TranslationController.getDateLangModule(newLocale);
+    this.format =
+      this.format || this.langModule?.formatLong?.time({ width: 'short' });
+    this.minTime = isMatch(this.minTime, this.format)
+      ? this.minTime
+      : format(startOfDay(new Date()), this.format);
+    this.maxTime = isMatch(this.maxTime, this.format)
+      ? this.maxTime
+      : format(endOfDay(new Date()), this.format);
```

Comment:
I think, for format, it expects the lang module should also be passed as argument

Reply:
The max and min time are used internally, both these values are converted into milliseconds for setting the upper and lower limit. Also the third is an argument to `format` function is optional.